tests/Util/ComposerAutoloaderFinderTest.php

```diff
@@ -59,6 +59,21 @@ public function testGetClassLoaderWhenItIsEmpty()
         (new ComposerAutoloaderFinder(static::$rootNamespace))->getClassLoader();
     }
 
+    public function testGetClassLoaderAfterEnabledSymfonyDebug()
+    {
+        if (\class_exists('Symfony\Component\Debug\Debug')) {
+            \Symfony\Component\Debug\Debug::enable();
+        }
+
+        if (\class_exists('Symfony\Component\ErrorHandler\DebugClassLoader')) {
+            \Symfony\Component\ErrorHandler\DebugClassLoader::enable();
+        }
+
+        $loader = (new ComposerAutoloaderFinder(static::$rootNamespace))->getClassLoader();
```

Comment:
After running the tests, I see (in red) `THE ERROR HANDLER HAS CHANGED`. We need to "replace" the error handlers at the end of the test - whether it fails or not. Probably at the start we need something like:

```
$handler = set_error_handler('var_dump');
$handler = \is_array($handler) ? $handler[0] : null;
restore_error_handler();
```

That is (I believe) the sort of "hacky" way to get a reference to the current error handler. Then, at the end, we need to `set_error_handler($handler)` to "replace" it. We need to do that if the test passes or is successful.

Reply:
Waiting for green from CI